Weather/Models/WeatherData.swift

```diff
@@ -2,17 +2,52 @@
 //  WeatherData.swift
 //  Weather
 //
-//  Created by Alex on 26.07.22.
+//  Created by Alex on 31.07.22.
 //
 
 import Foundation
 
-struct WeatherData {
-    let currentTemperature: Double
-    let humidity: Int
+struct WeatherData: Codable, Identifiable {
+    var dt: Int
+    var temp: Double
+    var feels_like: Double
+    var pressure: Int
+    var humidity: Int
+    var dew_point: Double
+    var clouds: Int
+    var wind_speed: Double
+    var wind_deg: Int
+    var weather: [WeatherDetail]
     
-    init(networkData: NetworkWeatherData) {
-        self.currentTemperature = networkData.current.temperature
-        self.humidity = networkData.current.humidity
+    enum CodingKey: String {
+        case dt
+        case temp
+        case feels_like
+        case pressure
+        case humidity
+        case dew_point
+        case clouds
+        case wind_speed
+        case wind_deg
+        case weather
+    }
+    
+    init() {
+        dt = 0
+        temp = 0.0
+        feels_like = 0.0
+        pressure = 0
+        humidity = 0
+        dew_point = 0.0
+        clouds = 0
+        wind_speed = 0.0
+        wind_deg = 0
+        weather = []
+    }
```

Comment:
The same about empty values

Reply:
Because i want be able to change default data if I needed